doc/api/repl.md

```diff
@@ -2,80 +2,129 @@
 
     Stability: 2 - Stable
 
-A Read-Eval-Print-Loop (REPL) is available both as a standalone program and
-easily includable in other programs. The REPL provides a way to interactively
-run JavaScript and see the results.  It can be used for debugging, testing, or
-just trying things out.
+The `repl` module provides a Read-Eval-Print-Loop (REPL) implementation that
+is available both as a standalone program or includable in other applications.
+It can be accessed using:
 
-By executing `node` without any arguments from the command-line you will be
-dropped into the REPL. It has simplistic emacs line-editing.
+```js
+const repl = require('repl');
+```
+
+## Design and Features
+
+The `repl` module exports the `repl.REPLServer` class. While running, instances
+of `repl.REPLServer` will accept individual lines of user input, evaluate those
+according to a user-defined evaluation function, then output the result. Input
+and output may be from `stdin` and `stdout`, respectively, or may be connected
+to any Node.js [stream][].
+
+Instances of `repl.REPLServer` support persistent history of prior inputs,
```

Comment:
That is true, it was discussed and decided against for increasing the API surface a few months ago.